src/components/Create/hooks/DeployProject/useDeployProject.ts

```diff
@@ -219,13 +223,22 @@ export const useDeployProject = () => {
         } else {
           deleteIfExists(projectMetadata, 'softTargetAmount')
         }
+
+        // Set domain to juicecrowd if juicecrowd project
+        if (
+          projectMetadata.softTargetAmount &&
+          (projectMetadata.introVideoUrl || projectMetadata.introImageUri)
+        ) {
+          domain = JUICECROWD_DOMAIN
```

Comment:
**suggestion**: could just use the feature flag to set the defaultState

Reply:
Thought about that, but theres a chance someone forgets. it feels unexpected to do this